src/keygenerator/main.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do we have an issue saying that this could/should be removed entirely?

Reply:
We now do: https://github.com/microsoft/CCF/issues/322 😄